images/win/scripts/Installers/Windows2019/Install-VS2019.ps1

```diff
@@ -72,6 +72,7 @@ $WorkLoads = '--allWorkloads --includeRecommended ' + `
               '--add Microsoft.VisualStudio.Component.VC.v141.MFC.ARM.Spectre ' + `
               '--add Microsoft.VisualStudio.Component.VC.v141.MFC.ARM64.Spectre ' + `
               '--add Microsoft.VisualStudio.Component.VC.v141.MFC.Spectre ' + `
+              '--add Microsoft.VisualStudio.Component.VC.14.25.x86.x64 ' + `
```

Comment:
Let's add Microsoft.VisualStudio.Component.VC.14.25.x86.x64.Spectre as well to be consistent

Reply:
Why we need to install `Microsoft.VisualStudio.Component.VC.14.25.x86.x64.Spectre` if it is not required in initial request?

Reply:
Just in case, because we've faced issues with missing Spectre libs before. However, I've checked the size and it's about 2Gb, so yeah, we can skip them for now.